com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs

```diff
@@ -11,19 +11,19 @@ namespace UnityEngine.Rendering
     internal class ProbeBrickIndex
     {
         // a few constants
-        internal const int kMaxSubdivisionLevels = 15; // 4 bits
+        internal const int kMaxSubdivisionLevels = 7; // 4 bits
```

Comment:
This can now be 3 bits, so at the very least the comment is wrong.

Reply:
Yep, though I'm not sure how is this comment useful 😅

Reply:
It is relevant because the subdivision level is packed in the index, so we need to know how many bits are used for that